packages/react-vapor/rollup.config.js

```diff
@@ -101,68 +101,68 @@ export default {
         'redux',
         'underscore',
         'coveo-styleguide',
-        'underscore.string'
+        'underscore.string',
     ],
     onwarn,
     plugins: [
         inject({
-            jQuery: 'jquery' // chosen-js expects jQuery to be available as a global
+            jQuery: 'jquery', // chosen-js expects jQuery to be available as a global
         }),
         replacePlugin(),
         postcss({
             extract: false,
             modules: {localIdentName: '[name]-[local]-[hash:base64]'},
             namedExports: true,
-            use: ['sass']
+            use: ['sass'],
         }),
         scssVariable(),
         resolve({
-            browser: true
+            browser: true,
         }),
         commonjs({
             namedExports: {
                 'hogan.js': ['Template', 'compile'],
                 'react-modal': ['setAppElement'],
                 'react-dnd': ['DragDropContext', 'DropTarget', 'DragSource'],
-                diff2html: ['diffChars', 'diffWordsWithSpace']
-            }
+                'diff/dist/diff.js': ['diffChars', 'diffWordsWithSpace'],
```

Comment:
I don't know why my pre-commit insist on formatting this file, but here is the only line that I changed in this PR.

Reply:
Same here but the other way around. Each time I commited in this file the formatter removed the commas.